lib/vanagon/platform/defaults/ubuntu-25.04-amd64.rb

```diff
@@ -0,0 +1,33 @@
+platform "ubuntu-25.04-amd64" do |plat|
+  plat.servicedir "/lib/systemd/system"
+  plat.defaultdir "/etc/default"
+  plat.servicetype "systemd"
+  plat.codename "noble"
+
+  packages = %w(
+    autoconf
+    build-essential
+    cmake
+    curl
+    debhelper
+    devscripts
+    fakeroot
+    libbz2-dev
+    libreadline-dev
+    libselinux1-dev
+    openjdk-8-jre-headless
+    pkg-config
+    quilt
+    rsync
+    swig
+    systemd
+    systemtap-sdt-dev
+    zlib1g-dev
+  )
+  plat.provision_with "export DEBIAN_FRONTEND=noninteractive; apt-get update -qq; apt-get install -qy --no-install-recommends #{packages.join(' ')}"
+  #plat.provision_with "curl https://apt.puppet.com/DEB-GPG-KEY-puppet-20250406 | apt-key add -"
```

Comment:
Might as well remove this

Reply:
Also, would you mind adding an aarch64 one as well? Don't worry about testing it if you don't have the hardware to do so.  Can make tweaks later if needed.